src/LSTM/index.js

```diff
@@ -76,83 +74,86 @@ class LSTM {
     const temperature = +options.temperature || this.defaults.temperature;
     const results = [];
 
-    if (this.ready) {
```

Comment:
I think this is a bugfix - the previous behavior is that calling this method before the model is fully loaded would do nothing.

Reply:
yep, thanks for the catch!